packages.config

```diff
@@ -3,4 +3,6 @@
   <package id="NUnit" version="2.6.4" targetFramework="net40" />
   <package id="NUnit.Runners" version="2.6.4" />
   <package id="FsCheck" version="2.0.3" />
+  <package id="NUnit" version="3.0.0-beta-3" targetFramework="net45" />
```

Comment:
do we really want two different versions of NUnit? How about removing the old one?